packages/components/src/progress_line/index.jsx

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import styles from './index.module.scss';
+
+export const PROGRESS_LINE_WEIGHT_THIN = 'thin';
+export const PROGRESS_LINE_WEIGHT_NORMAL = 'normal';
+
+const ProgressLine = (props) => {
+  const { steps, current, weight, ...cleanProps } = props;
+  const className = clsx(styles.root, props.className, {
+    [styles.thin]: weight === PROGRESS_LINE_WEIGHT_THIN,
+  });
+
+  const chunks = Array.from({ length: steps }, (_, index) => (
+    <li
+      key={index} className={clsx(styles.step, {
```

Comment:
Even though I can't think of a better option for key, maybe it's worth prefixing it with some string? 

Not 100% sure how it'll behave if we'll have other components with same index. 

Plus, would also move the className into a variable outside of JSX

Reply:
maybe you could use `arrayOf` here? https://github.com/goodhood-eu/nebenan-helpers/blob/master/src/data/index.js#L154